### ISSUE TITLE
[Bug] New LDAP user login starts in 'Org missing' context

### ISSUE THREAD

**Issue by taishan69** [grafana/grafana#7856]
Please include this information:
- What Grafana version are you using?
4.1.2-1486989747
- What datasource are you using?
Elasticsearch 5.x
- What OS are you running grafana on?
RHEL 7
- What did you do?
Login with a new AD user via LDAP
- What was the expected result?
That the user would be in the context of the first organization they are tagged into via role sync
- What happened instead?
User starts in "Org missing" and has to change roles to a valid organization on first login

TL/DR; I have very 'simple' users that want to see pretty stuff.  My use case entails essentially building an ISP looking glass server for circuit metrics for around 400 customers at about 1000 sites.

**Comment by daniellee** [comment]
Is this the same as this issue #7789 ?

**Comment by taishan69** [comment]
Sorry yes it appears it is; my search terms apparently weren't on the spot finding it

**Comment by daniellee** [comment]
No problem! Closing this as a dupe of #7789 then.
